### PR TITLE
Release Radius v0.40

### PR DESCRIPTION
**Pull request by willdavsmith**
# Description

Release Radius v0.40

## Type of change

- This pull request is a minor refactor, code cleanup, test improvement, or other maintenance task and doesn't change the functionality of Radius (issue link optional).

## Contributor checklist
Please verify that the PR meets the following requirements, where applicable:

- [x] An overview of proposed schema changes is included in a linked GitHub issue.
- [x] A design document PR is created in the [design-notes repository](https://github.com/radius-project/design-notes/), if new APIs are being introduced.
- [x] If applicable, design document has been reviewed and approved by Radius maintainers/approvers.
- [x] A PR for the [samples repository](https://github.com/radius-project/samples) is created, if existing samples are affected by the changes in this PR.
- [x] A PR for the [documentation repository](https://github.com/radius-project/docs) is created, if the changes in this PR affect the documentation or any user facing updates are made.
- [x] A PR for the [recipes repository](https://github.com/radius-project/recipes) is created, if existing recipes are affected by the changes in this PR.

### CODE REVIEW

**Review by ytimocin — DISMISSED** [review]

File: docs/release-notes/v0.40.0.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+## Intro to Radius
+
+If you're new to Radius, check out our website, [radapp.io](https://radapp.io), for more information. Also visit our [getting started guide](https://docs.radapp.io/getting-started/) to learn how to install Radius and create your first app.
+
+## Highlights
+
+<!-- TALK TO THE PM TEAM ABOUT WHAT HIGHLIGHTS TO ADD HERE -->
+
+TODO: PMs
```
Comment (Reshrahim) [inline docs/release-notes/v0.40.0.md:15]:
```suggestion
### `rad CLI` terminal
We have resolved a usability issue where the rad CLI prompt screen was truncated if the user's terminal window was small. The rad CLI prompt terminal now automatically scales to match the size of the user's terminal window.
```

File: docs/release-notes/v0.40.0.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+## Breaking changes
+
+<!-- ADD ANY BREAKING CHANGES HERE, IF ANY -->
+
+TODO: PMs
```
Comment (Reshrahim) [inline docs/release-notes/v0.40.0.md:21]:
```suggestion
None
```

File: docs/release-notes/v0.40.0.md (hunk 1 above)
Comment (Reshrahim) [reply docs/release-notes/v0.40.0.md:15]: @willtsai/ @sylvainsf - I added a usability issue we fixed to the highlights. Anything else we want to summarize from the change log ?

**Review by sylvainsf — APPROVED** [review]
:shipit:

**Review by sk593 — APPROVED** [review]

**Pull request merged**